### PR TITLE
account applications gunk api definitions

### PR DESCRIPTION
**Pull request by ghazninattarshah**
Added account applications gunk API definitions and updated the customer-related fields to profiles API

### CODE REVIEW

File: v1/accountapplications/accountapplications.gunk [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+	// Account application is created
+	Requested
+
+	// Account application is accepted
+	Accepted
+)
+
+// Structure of user information
+type User struct {
+
```
Comment (nkristianto) [inline v1/accountapplications/accountapplications.gunk]: nit: you can remove this line

File: v1/accountapplications/accountapplications.gunk [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+	// User identifier used to login
+	UserID string `pb:"1" json:"user_id"`
```
Comment (nkristianto) [inline v1/accountapplications/accountapplications.gunk]: can we use ID instead

File: v1/accountapplications/accountapplications.gunk [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+
+	// Email of the user
+	Email string `pb:"2" json:"email"`
+
+	// Display name of the user
+	Username string `pb:"3" json:"username"`
+}
+
+// Structure of account application
```
Comment (nkristianto) [inline v1/accountapplications/accountapplications.gunk]: maybe the comment could be
// AccountApplication is a structure of account application

File: v1/accountapplications/accountapplications.gunk [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+	ProductCode       string                   `pb:"2" json:"product_code"`
+	User              User                     `pb:"3" json:"user"`
+	CustomerProfile   types.Profile            `pb:"4" json:"customer_profile"`
+	DateOfApplication time.Time                `pb:"5" json:"date_of_application"`
+	Status            AccountApplicationStatus `pb:"6" json:"status"`
+}
+
+// Structure of credit rating information
+type CreditRating struct {
+
```
Comment (nkristianto) [inline v1/accountapplications/accountapplications.gunk]: nit: also remove this line

**Review by jwangsadinata — APPROVED** [review]
LGTM.

**Pull request merged**